libmscore/style.cpp

```diff
@@ -402,6 +401,14 @@ static const StyleType styleTypes[] {
     { Sid::minEmptyMeasures,        "minEmptyMeasures",        QVariant(2) },
     { Sid::minMMRestWidth,          "minMMRestWidth",          Spatium(4) },
     { Sid::mmRestNumberPos,         "mmRestNumberPos",         Spatium(-1.5) },
+    { Sid::mmRestNumberMaskHBar,    "mmRestNumberMaskHBar",    QVariant(true) },
+    { Sid::multiMeasureRestMargin,  "multiMeasureRestMargin",  Spatium(1.2) },
+    { Sid::mmRestHBarThickness,     "mmRestHBarThickness",     Spatium(0.7) },
+    { Sid::mmRestHBarVStrokeThickness, "mmRestHBarVStrokeThickness", Spatium(0.2) },
+    { Sid::mmRestHBarVStrokeHeight, "mmRestHBarVStrokeHeight", Spatium(2.0) },
+    { Sid::oldStyleMultiMeasureRests, "oldStyleMultiMeasureRests", QVariant(false) },
+    { Sid::mmRestOldStyleMaxMeasures, "mmRestOldStyleMaxMeasures", QVariant(9) },
+    { Sid::mmRestOldStyleSpacing,   "mmRestOldStyleSpacing",   Spatium(0.85) },
```

Comment:
Where did this value come from?  Looks wider than what Gould shows, and kind of arbitrary.

Reply:
Definitely arbitrary. It's the balance I came up with that seems to work well for all the music fonts. Gonville in particular has shockingly thick symbols that don't look good too close together. On the other hand, what do you think of making changing the font automatically change this setting, as well (which would give me an excuse to implement that TODO we talked about before)?

Reply:
Yes, per-font settings would be good.  I've never looked at how this is implemented for the other settings, but I take it is is based on some metadata the accompanies the font and is defined by SMuFL?  And the issue here is that there particular settings are not among those?  Since we have at least some time before MuseScore, I'd suggest actually making a proposal to the SMuFL folks to add what we need.

Reply:
Opened https://github.com/w3c/smufl/issues/125, but it looks like they've been inactive for some time.

Reply:
Well, I see a post from Daniel from February, and of course it's been a crazy time since then.  We'll see.  They've definitely made other changes at our request before though.

Reply:
On second thought, I think I'll leave this as is. It's wider than in Sibelius, but narrower than in Finale, and about the same as in Dorico and LilyPond.